src/test/resources/sya/allDetailsWithAppointeeWithSameAddress.json

```diff
@@ -0,0 +1,98 @@
+{
+  "benefitType": {
+    "description": "Personal Independence Payment",
+    "code": "PIP"
+  },
+  "mrn": {
+    "dwpIssuingOffice": "1",
+    "date": "01-02-2018"
+  },
+  "isAppointee": true,
+  "appellant": {
+    "title": "Mr",
+    "firstName": "Joe",
+    "lastName": "Bloggs",
+    "dob": "26-02-1990",
+    "nino": "AB877533C",
+    "isAddressSameAsAppointee": true,
+    "contactDetails": {
+      "phoneNumber": "07411222222",
+      "emailAddress": "sscstest@greencroftconsulting.com"
```

Comment:
I presume these contact details are always the same as the appointee for this scenario. Is that correct?

Why would the appellant enter different contact details if they didn't want notifications?